[DebugInfo] Use human-friendly printing for DWARF column attributes

jryans opened the pull request:
This prints DWARF attributes that describes source columns as base 10 integers for easier comprehension (via `llvm-dwarfdump` and similar tools) and matches the behaviour for source line attributes.

jryans committed:
[DebugInfo] Use human-friendly printing for DWARF column attributes

This prints DWARF attributes that describes source columns as base 10 integers
for easier comprehension (via `llvm-dwarfdump` and similar tools) and matches
the behaviour for source line attributes.

OCHyams commented:
LGTM. It looks like the test coverage for `DW_AT_call_line` printing is incidental (i.e., testing other things, such as in `test/tools/llvm-dwarfdump/X86/verify_attr_file_indexes_no_files.yaml`. Even so, if its easy to hijack or copy a test touching  `DW_AT_call_line` so that `DW_AT_{decl/call}_column` output can be tested too then we should probably do so.

jryans committed:
Update existing tests

jryans committed:
Add new source coordinates test

jryans commented:
Thanks for taking a look @OCHyams!

I have updated existing tests to match the new output and also added a new test specifically for checking this kind of source coordinate output.

adrian-prantl reviewed (approved):
I'm sure this is going to break _somebody's_ perl script, but I think that's a very reasonable change!
